accepted/0000-salt-new-maintaining-workflow.md

```diff
@@ -0,0 +1,300 @@
+
+### OBS project structure
+
+#### `systemsmanagement:saltstack:github/salt`
+
+This OBS package will only contain only a `_service` file that should look like:
+
+```
+<services>
+  <service name="obs_scm">
+    <param name="url">https://github.com/openSUSE/salt.git</param>
+    <param name="scm">git</param>
+    <param name="versionformat">@PARENT_TAG@</param>
+    <param name="versionrewrite-pattern">v(.*)</param>
+    <param name="revision">openSUSE/release/xxxx</param>
+    <param name="extract">pkg/suse/_multibuild</param>
+    <param name="extract">pkg/suse/salt.spec</param>
+    <param name="extract">pkg/suse/changelogs/factory/salt.changes</param>
+  </service>
+  <service name="set_version" />
+  <service name="tar" mode="buildtime"/>
+  <service name="recompress" mode="buildtime">
+    <param name="file">*.tar</param>
+    <param name="compression">gz</param>
+  </service>
+</services>
+```
+
+The rest of the files will be automatically pulled by the service, as they are enabled here. This package will be automatically refreshed by OBS at any new commit at `openSUSE/release/xxxx` branch.
+
+NOTE: This package will be automatically refreshed by OBS at any new commit at `openSUSE/release/xxxx` branch.
+
+#### `systemsmanagement:saltstack/salt`
+
+This is our ready-to-consume OBS package. We set the devel package (osc setdevelpackage) for this OBS package to `systemsmanagement:saltstack:github/salt`, but here we disable the services and manually run them to get the spec file, changelog and obsinfo/obscpio files, so the package can be submitted to openSUSE or SLE.
+
+The `_service` file here should look like:
+
+```
+<services>
+  <service name="obs_scm" mode="manual">
+    <param name="url">https://github.com/openSUSE/salt.git</param>
+    <param name="scm">git</param>
+    <param name="versionformat">@PARENT_TAG@</param>
+    <param name="versionrewrite-pattern">v(.*)</param>
+    <param name="revision">openSUSE/release/xxxx</param>
+    <param name="extract">pkg/suse/_multibuild</param>
+    <param name="extract">pkg/suse/salt.spec</param>
+    <param name="extract">pkg/suse/changelogs/factory/salt.changes</param>
+  </service>
+  <service name="set_version" mode="manual" />
+  <service name="tar" mode="buildtime"/>
+  <service name="recompress" mode="buildtime">
+    <param name="file">*.tar</param>
+    <param name="compression">gz</param>
+  </service>
+</services>
+```
+
+And it should only contain the following files:
+
+```
+_multibuild
+_service
+salt-xxxx.obscpio
+salt.changes
+salt.obsinfo
+salt.spec
+```
+
+Since services are disabled here, to allow submissions to openSUSE and SLE, this OBS package will be automatically synced with `openSUSE/release/xxxx` by a Jenkins job.
```

Comment:
Something is unclear about the workflow to me. You write that we use `systemsmanagement:saltstack:github/salt` as the devel project. Does that mean the changes go "Github" -> `systemsmanagement:saltstack:github/salt` -> `systemsmanagement:saltstack/salt` -> `openSUSE:Factory`? What exactly is this Jenkins job doing?